spring-rabbit/src/main/java/org/springframework/amqp/rabbit/listener/adapter/DelegatingInvocableHandler.java

```diff
@@ -184,6 +185,7 @@ private String extractSendTo(String element, SendTo ann) {
 
 	private String resolve(String value) {
 		if (this.resolver != null) {
+			value = this.beanExpressionContext.getBeanFactory().resolveEmbeddedValue(value);
```

Comment:
Does this work for you, Gary?

Also pay attention to the test changes.

Reply:
Sorry - I missed that; yes; it's ok with me - not worth pulling out to a utility (maybe next time if we need a change).

Reply:
Waiting for green 😄

Reply:
😃 . Right, just was going to same the same.

Not sure though that it is going to happen... I mean more changes in that part 😉